<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<include file="$(find ur_e_description)/launch/view_ur5e.launch"/>

	<rosparam ns="master" command="load" file="$(find vkr_sim)/config/setup.yaml" />
  <arg name="model" default="$(find ur_e_description)/urdf/ur5e.urdf"/>
  <param name="ur_urdf_model" command="$(find xacro)/xacro --inorder $(arg model)" />

  <node pkg="vkr_sim" name="vkr_test" type="vkr_test" output="screen"/>
</launch>
